src/plugins/intel_npu/thirdparty/CMakeLists.txt

```diff
@@ -30,8 +30,8 @@ if(ENABLE_ZEROAPI_BACKEND)
         endif()
     endif()
     set(CMAKE_COMPILE_WARNING_AS_ERROR OFF)
-    add_subdirectory(level-zero EXCLUDE_FROM_ALL)
 
-    # This VERSION file created by L0 may cause compilation issue of oneTBB headers, so remove it
-    file(REMOVE "${CMAKE_BINARY_DIR}/VERSION")
```

Comment:
looks like we can drop all this code above

Reply:
Removed